index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Microverse's Little Survey</title>
+</head>
+<body>
+  
+  <h1 id="title">How Do You Know Microverse?</h1>
+  
+  <p id="description">This form help us to know how do you know about Microverse</p>
+  
+  <form id="survey-form">
+
+<label>Name 
+  <input type="text" id="name" name="name" required placeholder="Enter your name" />
+</label>
+
+  <label for="email" id="email-label">Email</label>
+  <input id="email" type="email" required placeholder="Enter your email"/>
+  <label for="number" id="number-label"> Age (optional)</label>
+  <input id="number" type="number" min="1" max="10" required placeholder="Age">
+  <label for="select" id="Sex">Sex</label>
```

Comment:
This portion is satisfactory, with a positive aspect. Nevertheless, to enhance the readability of your code, I would recommend adopting the following approach for each label tag within this section:
`<label>
  Name
  <input type="text" id="name" name="name" required placeholder="Enter your name" />
</label>
`